vmm/src/lib.rs

```diff
@@ -605,58 +574,73 @@ impl EpollContext {
             epoll::ControlOptions::EPOLL_CTL_DEL,
             libc::STDIN_FILENO,
             epoll::Event::new(epoll::Events::EPOLLIN, self.stdin_index),
-        )
-        .map_err(Error::EpollFd);
+        );
```

Comment:
If errors are ignored and this function always returns Ok(()) then why do we have the return value of disable_stdin_event a `Result`?

Reply:
Indeed, also a similar issue holds for enable_stdin_event. Moreover, once these two functions are made not to return anything, rather than returning results, the function enable_disable_stdin_test is made useless (assuming that epoll::ctl cannot panic), so I've removed it too.